libs/ledger-live-common/src/families/avalanchepchain/js-transaction.ts

```diff
@@ -0,0 +1,30 @@
+import { BigNumber } from "bignumber.js";
+import type { Account } from "@ledgerhq/types-live";
+import type { Transaction } from "./types";
+
+const createTransaction = (): Transaction => ({
+  family: "avalanchepchain",
+  amount: new BigNumber(0),
+  recipient: "",
+  useAllAmount: false,
+  fees: null,
+  mode: "delegate",
+  startTime: null,
+  endTime: null,
+  maxEndTime: null,
+});
+
+const prepareTransaction = async (
+  account: Account,
+  transaction: Transaction
+) => {
+  transaction.amount = transaction.useAllAmount
+    ? account.spendableBalance
+    : transaction.amount;
+
+  return transaction;
```

Comment:
Just to confirm, no fees are required to perform a delegate? Transactions with no fees will always succeed?

Reply:
Correct! There are no fees required to delegate on the p chain.

Reply:
that answer my question above then 🙄